src/senaite/sync/importstep.py

```diff
@@ -133,7 +133,7 @@ def _import_data(self):
                 logger.info("Committed: {} / {} ".format(
                             self._non_commited_objects, len(ordered_uids)))
                 self._non_commited_objects = 0
-
+            logger.info("Imported: {} / {}".format(item_count+1, len(ordered_uids)))
```

Comment:
You are calculating here the length for each item in the count. You can speed up things by defining a variable outside of the loop.